apps/api/env/.env.functional.test

```
@@ -1,4 +1,6 @@
 POSTGRES_URI=postgres://postgres:password@localhost:5432
+SEQUELIZE_POOL_MIN=1
+SEQUELIZE_POOL_MAX=1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SEQUELIZE_POOL_MAX key should go before the SEQUELIZE_POOL_MIN key

(UnorderedKey)